prospero-cli/src/main/java/org/wildfly/prospero/cli/commands/InstallCommand.java

```diff
@@ -103,6 +103,17 @@ public Integer call() {
             return ReturnCodes.INVALID_ARGUMENTS;
         }
 
+        if (directory.toFile().isFile()) {
```

Comment:
Can you add those checks to `org.wildfly.prospero.actions.Provision` as well? This should be enforced if Prospero is run via GUI as well

Reply:
Good point. I would rather move them to the common module, than duplicate. I will need to add some message logger (that can be localized) to the common module for that as well.

Reply:
Alternatively the Provision can throw an exception the CLI will interpret and show appropriate message

Reply:
You can check it out @spyrkob. I used jboss-logging to handle message bundles. We could now add `Messages.i18n_${lang}.properties` files to `resources/org/wildfly/prospero/` to provide language translations.

Any `IllegalArgumentException` that comes from `Provision` class would be treated as wrong argument error.

Reply:
We could rework the `Messages` class in `prospero-cli` in similar fashion.